### PR TITLE
EZP-27780: Remove ezsearch_return_count table

### PR DESCRIPTION
**Pull request by wizhippo**
Relates to [EZP-23082](https://jira.ez.no/browse/EZP-23082): completely remove ezsearch_return_count table

### CODE REVIEW

**Comment by andrerom** [comment]
Nice catch :)

Besides needing a new issue since there has been releases in between, will also need to be dropped from https://github.com/ezsystems/ezpublish-kernel/blob/master/data/mysql/schema.sql#L1904 and we'll need to copy/adapt update sql script here: https://github.com/ezsystems/ezpublish-kernel/tree/master/data/update

**Comment by andrerom** [comment]
@wizhippo can you rebase for 6.7?

File: data/update/mysql/dbupdate-5.3.0-to-5.4.0.sql [hunk 1]
```diff
@@ -0,0 +1 @@
+DROP TABLE ezsearch_return_count;
```
Comment (andrerom) [inline data/update/mysql/dbupdate-5.3.0-to-5.4.0.sql]: One last thing, platform users will need to drop this during an upgrade, so move this and the other one to`dbupdate-6.10.0-to-6.11.0.sql` 

And add inline comment like `--eZ Publish users who has upgraded will get warning that table was missing, this is fine as you'll already removed it when upgrading to 5.4`

**Comment by andrerom** [comment]
@alongosz Reason for suggesting 6.7 was to avoid issues for people coming from legacy, as it implies Platform code refer to a table they don't have.

However I can skip the db update script as part of 6.7 on the merge if that makes it more clean for you?

**Comment by alongosz** [comment]
> However I can skip the db update script as part of 6.7 on the merge if that makes it more clean for you?

Yes, please :)

**Comment by andrerom** [comment]
Merged in:
- https://github.com/ezsystems/ezpublish-kernel/commit/f6f58f246d06b96524e157341b12124d1d731b53 (6.7 and up)
- https://github.com/ezsystems/ezpublish-kernel/commit/8d723b7fff6a1212e6c377b393144e759f107aed (sql upgrade script, 6.11 and up)

**Pull request closed without merging**